cedar-policy/src/frontend/is_authorized.rs

```diff
@@ -228,62 +277,62 @@ struct RecvdSlice {
     template_instantiations: Option<Vec<TemplateLink>>,
 }
 
-impl RecvdSlice {
```

Comment:
Is all this whitespace change to split up the function for the clippy lint?

Reply:
To make it more readable, it was a bunch of nested functions.